src/controllers/accounts.go

```diff
@@ -0,0 +1,54 @@
+package controllers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/rodolfoalvesg/api-banking/api/src/entities/accounts"
+	"github.com/rodolfoalvesg/api-banking/api/src/models"
+	"github.com/rodolfoalvesg/api-banking/api/src/responses"
+)
+
+// CreateAccount cria uma conta
+func (c *Controller) HandlerCreateAccount(w http.ResponseWriter, r *http.Request) {
+	var account models.Account
+
+	if err := json.NewDecoder(r.Body).Decode(&account); err != nil {
+		responses.RespondError(w, http.StatusBadRequest, err)
+	}
+
+	acc, err := accounts.CreateAccount(account)
+	if err != nil {
+		responses.RespondJSON(w, http.StatusBadRequest, err)
+		return
+	}
+
+	defer r.Body.Close()
+
+	responses.RespondJSON(w, http.StatusOK, acc)
+}
+
+// ShowBalance, exibe o saldo
+func (c *Controller) HandlerShowBalance(w http.ResponseWriter, r *http.Request) {
+	params := mux.Vars(r)
+	accountId := params["account_id"]
```

Comment:
Caso o parâmetro não exista, deveria ser possível continuar? Caso contrário é bom retornar um erro aqui!

Reply:
Outro ponto é que, em Go, algumas abreviações são escritas em CAPS! Ou seja:
```suggestion
	accountID := params["account_id"]
```

Reply:
É vou ter que alterar em outros lugares. Acho que havia lido sobre, porém acabei esquecendo essa regrinha.

Reply:
> Caso o parâmetro não exista, deveria ser possível continuar? Caso contrário é bom retornar um erro aqui!

Aqui... eu coloquei no Showbalance do entities para caso o params venha com valor vazio eles retorna erro,

Reply:
É melhor deixar no handler mesmo essa verificação?